src/css/app.scss

```diff
@@ -30,6 +30,15 @@ body {
     flex-shrink: 0;
 }
 
+/* Styling for closing button on the grid items */
+.close-pane-button {
+    background: none;
+    border: none;
+    font-family: Ampersand;
```

Comment:
When I disable this rule, rather than changing the font, the color of the text changes, to a light grey... And conversely, I can remove the `color` below without any ill effects. Do you understand what's going on?